website/docs/r/guardduty_ipset.html.markdown

```diff
@@ -0,0 +1,53 @@
+---
+layout: aws
+page_title: 'AWS: aws_guardduty_ipset'
+sidebar_current: docs-aws-resource-guardduty-ipset
+description: Provides a resource to manage a GuardDuty IPSet
+---
+
+# aws_guardduty_ipset
+
+Provides a resource to manage a GuardDuty detector.
```

Comment:
Copypasta: Should be `GuardDuty IPSet`

Reply:
We might want to include the note from the [GuardDuty API documentation](https://docs.aws.amazon.com/guardduty/latest/ug/create-ip-set.html) here.
```
~> **Note:** Currently in GuardDuty, users from member accounts cannot upload and further manage IPSets. IPSets that are uploaded by the master account are imposed on GuardDuty functionality in its member accounts.
```